vmprof/__init__.py

```diff
@@ -36,14 +43,41 @@ def enable(fileno, period=DEFAULT_PERIOD, memory=False, warn=True):
         if warn and sys.pypy_version_info[:3] < (4, 1, 0):
             print ("PyPy <4.1 have various kinds of bugs, pass warn=False if you know what you're doing")
             raise Exception("PyPy <4.1 have various kinds of bugs, pass warn=False if you know what you're doing")
-        _vmprof.enable(fileno, period)
+        gz_fileno = _gzip(fileno)
+        _vmprof.enable(gz_fileno, period)
 
     def disable():
         _vmprof.disable()
+        _gzip_finish()
 
     def enable_jitlog(fileno):
         """ Should be a different file than the one provided
             to vmprof.enable(...). Otherwise the profiling data might
             be broken.
         """
-        _vmprof.enable_jitlog(fileno)
+        gz_fileno = _gzip(fileno)
+        _vmprof.enable_jitlog(gz_fileno)
+
+
+_gzip_procs = []
+
+def _gzip(fileno):
```

Comment:
above you use the return value of this function. But it always returns None? Is that tested?